__tests__/pages/profile/username.test.js

```diff
@@ -38,14 +38,13 @@ describe('user profile test', () => {
         }
       }
     ]
-    const { container, findByRole, queryByText } = render(
+    const { findByRole, queryByText } = render(
       <MockedProvider mocks={mocks} addTypename={false}>
         <UserProfile />
       </MockedProvider>
     )
     await findByRole('heading', { name: /loading/i })
     await waitForElementToBeRemoved(() => queryByText('Loading...'))
-    expect(container).toMatchSnapshot()
```

Comment:
I deleted this snapshot. It's flaky (it fails if you pc is under heavy load). It also unnecessary since I only need to check for loading spinner.